restart_plugin/mtcp_restart_plugin.c

```diff
@@ -340,6 +340,27 @@ load_mana_header (char *filename, ManaHeader *mh)
   return 0;
 }
 
```

Comment:
I think a better name is set_header_fikepath().
In addition, let's place the OUT argument first: full_filename,
and optionally (not required). add a comment,:
//!IN: ...; OUT: ...